cluster/manager.go

```diff
@@ -867,15 +869,15 @@ func (c *ClusterManager) Start() error {
 	// Start heartbeating to other nodes.
 	go c.startHeartBeat(clusterInfo)
 
-	c.startClusterDBWatch(initState, kvdb)
+	c.startClusterDBWatch(lastIndex, kvdb)
 
 	err = c.waitForQuorum(exist)
 	if err != nil {
 		return err
 	}
 
 	go c.updateClusterStatus()
-	go c.replayNodeDecommission(initState)
+	go c.replayNodeDecommission(clusterInfo)
```

Comment:
cluster info is stale, remove it as a param here and instead do readClusterInfo() as a part of replayNodeDecommision (or i can do it as a part of my clean up too).